arrows/ocv/resection_camera.cxx

```diff
@@ -205,15 +202,42 @@ ::resection(
 
   auto const reproj_error = d_->reproj_accuracy;
 
-  auto const err =
-    cv::calibrateCamera(
+  auto err = std::numeric_limits<double>::infinity();
+  vital::matrix_3x3d K = cal->as_matrix();
+  cv::TermCriteria term_criteria{
+    cv::TermCriteria::COUNT + cv::TermCriteria::EPS,
+    d_->max_iterations, DBL_EPSILON };
+  using MatD = cv::Mat_<double>;
+  MatD cv_K;
+  eigen2cv( K, cv_K );
+  auto const dc0 = dist_coeffs;
+  auto const focal_scales = {0.5, 1.0, 2.0};
```

Comment:
Are these three scale always sufficient?  Not critical, but it might be nice to make this list a config parameter at some point (doesn't need to hold up this PR).

Reply:
those values appear to be sufficient for the projects where we use manual calibration; having them in a configuration file may indeed be a good idea; I may need some guidance on how to represent a vector of floats in a config file